components/pipeline-service/production/stone-prod-p01/resources/tekton-chains-signing-secret-path.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/dataFrom/0/extract/key
+  value: staging/pipeline-service/stone-stage-p01/chains-signing-secret
```

Comment:
wrong key path? there is stage in the name of the secret

Reply:
Yup, I forgot to save the file :disappointed: 
Fixed.